### PR TITLE
(Review Completed) Cheeky trickster proofreading

### PR DESCRIPTION
**Pull request by HotHeadEmperor**
Small change to Eruru file, too. Noticed during playtesting her 'Beginning of Duel' line mentioned it was 'her turn' no matter whose turn it actually was.

### CODE REVIEW

File: CSV/Language/English/Card/10017.txt [hunk 1]
```diff
@@ -3,31 +3,31 @@ default,
 EU17_paizuriAnime,
 盗みが得意な女の子。胸をわざとらしく揺らしながら歩いて男性の視線を誘導し、その隙にポケットから大事なものを盗んでいく。,A girl who excells at stealing. She guides male gazes to the intentional sway of her breasts while she walks; giving her ample opportunity to reach into their pockets and steal their valuables.
 わたしの大きいおっぱいで気持ちよくなっちゃう？\nほら、こんな風におっぱいで挟んで…❤,Are my large breasts making you feel good? Hora; I'll wrap your dick up between them just like this...❤
-むぎゅーってしてあげれば、たまんないっしょ？\nあはは、腰浮かせちゃって、素直ー❤,When I give my tits a big squeeze; you lose control over your dick right? Ahaha! The way your hips suddenly buck upwards shows your honest feelings about me~❤
-わたし、おっぱいには自信あってさ、\n上級モンスター相手でもこのおっぱいで勝てちゃうことあるんだよね。,I'm very good with my breasts; to the point I can exclusively use them to win against advanced class monsters~.
+むぎゅーってしてあげれば、たまんないっしょ？\nあはは、腰浮かせちゃって、素直ー❤,When I give my tits a big squeeze; you lose control over your dick; right? Ahaha! The way your hips suddenly buck upwards shows your honest feelings about me~❤
```
Comment (Giratena) [inline CSV/Language/English/Card/10017.txt:6]: Very good reword here!

File: CSV/Language/English/Card/10017.txt [hunk 2]
```diff
@@ -3,31 +3,31 @@ default,
...
-そんなに我慢できなかったわけ？\nさすがは低級ランクのマスターだね❤理性弱すぎ❤,You can't even stand it for that long? As expected from a low class cardmaster.❤ Damn your willpower is weak.❤
-そんなに我慢できなかったわけ？\nさすがは低ランクマスターのモンスターだね❤理性弱すぎ❤,You can't even stand it for that long? As expected from the monster from a low class cardmaster.❤ Damn your willpower is weak.❤
-そんなに我慢できなかったわけ？かっこわる❤\nサキュバスに勝ったくせにわたしには負けるわけ？理性弱すぎ❤,You can't even stand it for that long? Damn.❤ You beat a succubus and you're losing to me? Damn your willpower is weak.❤
-そんなに我慢できなかったわけ？かっこわる❤\n男のモンスターなんてしょせんこんなもんか❤\nきゃははは❤理性よわーい❤,You can't even stand it for that long? Terrible.❤ Male monsters really are useless huh? Kyahaha!❤ Such weak willpower~❤
+戦いの最中なのにチンポ、ギンギンじゃん❤\nわたしの体見て、興奮しちゃった？,Even though we're in the middle of a fight; your dick is rock hard; ain't it?❤ Did you get turned on while staring at my body?
+それじゃほら、服を引っ張って谷間見せたげる❤\n見とれてるあいだに、アンタのチンポシコシコしちゃうね。\nほら、しこしこしこー❤,Well then; look here. I'll pull my shirt down and show you my cleavage.❤ And I'll just go ahead and stroke your dick while you stand there staring; OK? That's it. Stroke stroke stroke~❤
```
Comment (Giratena) [inline CSV/Language/English/Card/10017.txt:17]: I was thinking what word could have been better used here. You nailed it haha

File: CSV/Language/English/Card/10017.txt [hunk 3]
```diff
@@ -3,31 +3,31 @@ default,
...
-あは、わたしのおっぱいに夢中じゃん？それならほら…。,Aha! You're really focused on my boobs; aren't-cha? Well then; in that case...
+ぷっ、バカにされてるのに「おっぱいで挟んでぇ」とか言ってるし❤\nそんなにイキたいなら抜いてあげるよ❤\nあたしのパイズリに何秒耐えられるかな～❤,Pfft~ Even though I'm making fun of you; you're saying; "please wrap your boobs around my dick~"❤ If you're already about to cum; then I'll drain you dry~❤
+ぷっ、バカにされてるのに「手でしごいてぇ」とか言ってるし❤\nそんなにイキたいなら抜いてあげるよ❤\nガシガシ扱かれても悲鳴あげるなよ～❤,Pfft~ Even though I'm making fun of you; you're saying; "please stroke my dick~"❤ If you're already about to cum; then I'll drain you dry~❤
+ぷっ❤どんだけ飛ばしてんの？\n雑にシコられたぐらいでこんなに出して、チンポ弱すぎっしょ❤\nザーコ❤,Hah~❤ Man; it's really shooting out; huh? Your dick must really be weak to cum just from a short wank~❤ Small Fry!❤
+あはは❤ズリズリされて全然耐えられないじゃん❤\nあたしのおっぱいヤバいぐらい気持ちよかったっしょ？\n腰抜かしてガクガク痙攣してるし。どんだけだよ❤ザーコ❤,Ahaha!❤ You really can't resist at all when your dick is being rubbed beween my boobs; huh?❤ My breasts feel good; right? Your hips are trembling; too. You really are something.❤ Small Fry~❤
+あは、わたしのおっぱいに夢中じゃん？それならほら…。,Aha! You're really focused on my boobs; ain'tcha? Well then; in that case...
```
Comment (Giratena) [inline CSV/Language/English/Card/10017.txt:30]: Nice!

File: CSV/Language/English/Card/10017.txt [hunk 4]
```diff
@@ -37,62 +37,62 @@ EU17_paizuriAnime,
...
-あっという間に負けてくれてラッキーって感じ？谷間で負けてるようじゃ、この先ないよ？雑魚マスター❤くすくす❤,Do you feel lucky to lose to me? If you lost just by staring at my cleavage like this; there's no way you'll make it any further than this you know? Small fry Cardmaster~❤ *Giggle*❤
+あんっ、すごい勢い。おっぱいがどろどろじゃん。オナニーでどんだけ出すんだよ、変態❤,Ahn~ It's still flowing out. My boobs are such a mess now. Who would have thought a pervert would come along and creampie me?❤
+ほんと谷間に何回負けるの？雑魚すぎじゃん。おっぱいマゾの変態マスター❤ぷぷっ。,Seriously; how many times are you going to lose to my cleavage? You're a real masochist boob addict; Cardmaster~❤ Heheh.
+あっという間に負けてくれてラッキーって感じ？谷間で負けてるようじゃ、この先ないよ？雑魚マスター❤くすくす❤,Do you feel lucky to lose to me? If you lost just by staring at my cleavage like this; there's no way you'll make it any further; you know? Small Fry Cardmaster~❤ *Giggle*❤
 あっという間に負けてくれてラッキーって感じ？ほら、他の男もわたしでオナニーしていいんだよー？くすくす❤,Do you feel lucky to lose to me? Come on; any other guys who want to stroke their dick to me? *Giggle*❤
-あはは。ほんとに別のやつもオナニーしはじめちゃった❤,Ahaha! Someone else actually started masturbating as well.❤
+あはは。ほんとに別のやつもオナニーしはじめちゃった❤,Ahaha! Someone else is actually started masturbating as well.❤
```
Comment (Giratena) [inline CSV/Language/English/Card/10017.txt:53]: Revert this change. I think it was fine as is ^^

File: CSV/Language/English/Card/70006.txt [hunk 5]
```diff
@@ -26,7 +26,7 @@ Sexy01,
 はい、わたしの勝ち。やっぱりE級とD級じゃ、実力が違うわよね。クスクス。,Yep; looks like I won. There really is a big gap between E-rank and D-rank; *Giggle*
 うっ、さすがね。やるじゃない、{主人公}。\nでも次は勝つわ。またバトルしましょう。,Ugh; I thought so. You're not half bad; {主人公}. But next time; victory will be mine. Let's battle again sometime; OK?
 はい、わたしの勝ち❤\n気持ちよく射精できてよかったわね❤うふふ❤,Yep; looks like I won.❤ I bet you're glad to get such a pleasurable ejaculation.❤ Ufufu~❤
-あなたのターンからね。さあ、いいバトルをしましょう。,It's my turn. Let's have a good battle.
+あなたのターンからね。さあ、いいバトルをしましょう。,Match start. Let's have a good battle.
 わたしのターンからね。いくわよ、わたしの闇属性モンスターでこてんぱんにしてあげる。,It's my turn. My Dark Attribute monster will beat you thoroughly.
```
Comment (Giratena) [inline CSV/Language/English/Card/70006.txt:30]: This should be "Looks like it's my turn."

Also the last line sounds a little weird too. Maybe something like, "Let's go. My Dark Attribute monster is gonna mess you up good!"

File: CSV/Language/English/Card/70006.txt [hunk 6]
```diff
@@ -26,7 +26,7 @@ Sexy01,
 はい、わたしの勝ち。やっぱりE級とD級じゃ、実力が違うわよね。クスクス。,Yep; looks like I won. There really is a big gap between E-rank and D-rank; *Giggle*
 うっ、さすがね。やるじゃない、{主人公}。\nでも次は勝つわ。またバトルしましょう。,Ugh; I thought so. You're not half bad; {主人公}. But next time; victory will be mine. Let's battle again sometime; OK?
 はい、わたしの勝ち❤\n気持ちよく射精できてよかったわね❤うふふ❤,Yep; looks like I won.❤ I bet you're glad to get such a pleasurable ejaculation.❤ Ufufu~❤
-あなたのターンからね。さあ、いいバトルをしましょう。,It's my turn. Let's have a good battle.
```
Comment (Giratena) [inline CSV/Language/English/Card/70006.txt:29]: This means "Looks like it's your turn".  Not her turn ^^

**Review by Giratena — CHANGES_REQUESTED** [review]
Great review overall! You're getting pretty good at this!

Just some changes in Eruru's file and one change in Carol's file. Good job!

File: CSV/Language/English/Card/10017.txt (hunk 4 above)
Comment (HotHeadEmperor) [reply CSV/Language/English/Card/10017.txt:53]: Lol this is my bad. I read "starting" instead of "started." Will fix right away.

File: CSV/Language/English/Card/70006.txt (hunk 6 above)
Comment (HotHeadEmperor) [reply CSV/Language/English/Card/70006.txt:29]: Ooooooh. That makes sense. Okay.

File: CSV/Language/English/Card/10017.txt (hunk 4 above)
Comment (Giratena) [reply CSV/Language/English/Card/10017.txt:53]: No problem at all!
